modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -336,13 +331,12 @@
 			<description>
 				Rounds [code]s[/code] downward (towards negative infinity), returning the largest whole number that is not more than [code]s[/code].
 				[codeblock]
-				# a is 2.0
-				a = floor(2.99)
-				# a is -3.0
-				a = floor(-2.99)
+				a = floor(2.45)  # a is 2.0
+				a = floor(2.99)  # a is 2.0
+				a = floor(-2.99) # a is -3.0
 				[/codeblock]
 				See also [method ceil], [method round], and [method stepify].
-				[b]Note:[/b] This method returns a float. If you need an integer, you can use [code]int(s)[/code] directly.
+				[b]Note:[/b] This method returns a float. If you need an integer and [code]s[/code] is a non-negative number, you can use [code]int(s)[/code] directly.
```

Comment:
Good catch. I think we should also mention that `int()` is a truncation operation.

Reply:
I wrote this on the `int` page, added cross-references.